qemu/tests/block_with_iommu.py

```diff
@@ -13,12 +14,32 @@ def run(test, params, env):
         1. Add "intel_iommu=on" to kernel line of q35 guest.
```

Comment:
This is x86 only case, please ignore me.

Reply:
Sorry for inconvenience.